paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/CountrySettingsDefinition.kt

```diff
@@ -51,19 +51,22 @@ internal object CountrySettingsDefinition :
     override fun valueUpdated(value: Country, playgroundSettings: PlaygroundSettings) {
         // When the country changes via the UI, update the currency to be the default currency for
         // that country.
-        when (value) {
```

Comment:
The last item here was the straw that broke the camel's back and we [exceeded the CyclomaticComplexity](https://app.bitrise.io/build/439b7f14-e21b-4c37-8476-d76bab12c765) threshold for this method.

Moving this to a map should solve the issue.